modules/nf-core/mageck/test/nextflow.config

```diff
@@ -0,0 +1,12 @@
+process {
```

Comment:
the module directory itself doesn't need a nextflow.config (just the test dir)- so just delete this file.